src/app.ts

```diff
@@ -76,6 +77,25 @@ const TOKEN_DURATION_IN_SECONDS = Number(
   getRequiredEnvVar("TOKEN_DURATION_IN_SECONDS")
 );
 
+const emailService = new EmailService({
+  auth: {
+    pass: getRequiredEnvVar("EMAIL_PASSWORD"),
+    user: getRequiredEnvVar("EMAIL_USER")
+  },
+  from: getRequiredEnvVar("EMAIL_SENDER"),
+  host: getRequiredEnvVar("EMAIL_SMTP_HOST"),
+  port: Number(getRequiredEnvVar("EMAIL_SMTP_PORT")),
+  secure: getRequiredEnvVar("EMAIL_SMTP_SECURE") === "true"
+});
+
+emailService
+  .verifyTransport()
```

Comment:
this should be chained to the subsequent call to newApp() somewy otherwise they will run in parallel.
something like `verifyEmailService(..).then(() => newApp(...))`